### DISCUSSION TITLE
程序出了问题，是重头再来还是修复BUG，公司会用哪个方式？

### DISCUSSION THREAD

**Discussion by tingbiaolab** [WeNeedHome/SummaryOfLoanSuspension#1158] [General]
那么大体量的程序会不会重新编写啊！

**Comment by itsoursduty** [comment]
这个比喻不太恰当

**Comment by guDuShouHuZhe** [comment]


**Reply by xijinping0** [reply]
墙内程序员哪还有圣地，Github 早就慢得要死了。

**Reply by go2006010215** [reply]
不应该怪GFW吗？搁这儿堵人嘴？

**Reply by wedxz** [reply]
不怪杀人犯，怪起卖刀的了？

**Comment by oceanzhang860411** [comment]
這哪是bug？這程序原來就這麼設計的，第一次就是用現有程序覆蓋所有既有程序（打地主分田地&公私合營），然後程序沒有資源，宕機，第二次就搞bug清除，按比例劃分有問題的功能點（反右，文化大革命），然後所有功能點都失效了，又宕機
你現在跟我說繼續呀，我懷疑你是弱智

**Reply by tingbiaolab** [reply]
你要是会说话就好好讨论，不会说话就去猪圈用屎把嘴洗干净了再说话。长个脑仁就以为自己是天下无敌智者了。

**Reply by oceanzhang860411** [reply]
如果腦仁都沒有，那就別討論了，直接做奴隸不是更簡單？

**Reply by tingbiaolab** [reply]
> 如果腦仁都沒有，那就別討論了，直接做奴隸不是更簡單？

你那松子大的脑仁要是真有思考的话，就不会在这用个局外人的角色夸夸其谈了。

**Reply by oceanzhang860411** [reply]
看來你核酸還是做得不夠啊，能出小區了嗎？

**Reply by Ventery** [reply]
台巴子真是烂泥扶不上墙😅

**Reply by oceanzhang860411** [reply]
> 台巴子真是烂泥扶不上墙😅

來電了？能用電腦了？

**Reply by Cegazhos** [reply]
台湾狗怎么不说程式？侯捷老师的书不看是吧

**Reply by oceanzhang860411** [reply]
> 台湾狗怎么不说程式？侯捷老师的书不看是吧

“南京是一个50万人口的大城市，国民党的首都，应杀的反动分子似不止200余人……南京杀人太少，应在南京多杀！上海是一个600万人口的大城市，我认为1951年内至少应当杀掉罪大的……3000人左右。而在上半年至少应杀掉1500人左右。”——毛泽东《对上海南京镇反工作的指示》，1951年2月12日。
書建議多看，認真看

**Reply by Cegazhos** [reply]
> 对上海南京镇反工作的指示

经典答非所问 转移话题。其次，你个傻蛋 链接不发出版社不发 扣扣索索一段字 你让你爹怎么看？
最后，我又不像你一天到晚搞政治给自己找合法存在， 那我看这个干吊？

**Comment by xijinping0** [comment]
这哪里是bug，红色江山代代传，十里山路不换肩。一条道走到黑！

[![不换肩](https://img.youtube.com/vi/cdH-zKCQs1M/0.jpg)](https://youtu.be/cdH-zKCQs1M?t=2)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》

**Reply by tingbiaolab** [reply]
开个小号，哪里有事哪里到。

**Comment by Accelerator6666** [comment]
不是应该先评估么？评估完再根据情况决定整改规模

**Comment by Cegazhos** [comment]
董事会不会重构的。除非董事会有眼光去看长期收益
